apps/daimo-mobile/src/logic/note.ts

```diff
@@ -55,13 +55,12 @@ export function useFetchNote(
 async function fetchNote(
   ephemeralOwner: `0x${string}`
 ): Promise<EphemeralNote | undefined> {
-  const notesContract = getContract({
-    abi: Contracts.ephemeralNotesABI,
-    address: Contracts.ephemeralNotesAddress,
-    publicClient: getClientL2(),
+  const res = await getClientL2().readContract({
+    ...Contracts.ephemeralNotesConfig,
+    functionName: "notes",
+    args: [ephemeralOwner],
```

Comment:
i switched the style here from viem's contract object (via `getContract`) to directly calling `readContract`/`writeContract` for two reasons:

1. `getContract` Typescript types are overcomplicated in latest viem
2. Documentation says this can reduce bundle size by pulling in fewer dependencies. (The contract object has ~all JSON RPC methods on it.)